internal/expr/expr_test.go

```diff
@@ -43,6 +43,10 @@ type IntMap map[string]int
 
 type StringMap map[string]string
 
+type S []any
+type T []int
+type U []string
```

Comment:
These type names would benefit from some love. :)

Reply:
Changed the names to reflect the content of the slices.